src/app/map/services/esri-services/tool-service/strategies/drawing/esri-symbol-drawing.strategy.ts

```diff
@@ -0,0 +1,83 @@
+import {AbstractEsriDrawingStrategy} from '../abstract-esri-drawing.strategy';
+import {DrawingCallbackHandler} from '../../interfaces/drawing-callback-handler.interface';
+import {MatDialog} from '@angular/material/dialog';
+import {SupportedEsriTool} from '../supported-esri-tool.type';
+import {DrawingMode} from '../../types/drawing-mode.type';
+import {SymbolDrawingToolInputComponent} from 'src/app/map/components/symbols-drawing-tool-input/symbol-drawing-tool-input.component';
+import {PanelClass} from 'src/app/shared/enums/panel-class.enum';
+import {filter, tap} from 'rxjs';
+import Graphic from '@arcgis/core/Graphic';
+import {scaleCIMSymbolTo, applyCIMSymbolRotation} from '@arcgis/core/symbols/support/cimSymbolUtils.js';
+import CIMSymbol from '@arcgis/core/symbols/CIMSymbol';
+import {Gb3SymbolStyle} from 'src/app/shared/interfaces/internal-drawing-representation.interface';
+import WebStyleSymbol from '@arcgis/core/symbols/WebStyleSymbol';
+import {MapDrawingSymbol} from '../../../types/map-drawing-symbol.type';
+
+export type SymbolDrawingInputComponentOutput = {
+  webStyleSymbol: WebStyleSymbol;
+  size: number;
+  rotation: number;
+};
+
+export class EsriSymbolDrawingStrategy extends AbstractEsriDrawingStrategy<DrawingCallbackHandler['completeDrawing']> {
+  protected readonly tool: SupportedEsriTool = 'point'; // Technically. ESRI doesn't know a "symbol" drawing strategy. We have to emulate that. A symbol is itself set with a point geometry, so it _does_ check out.
+  private readonly dialogService: MatDialog;
+  private mapDrawingSymbol: MapDrawingSymbol | undefined = undefined;
+  private symbolSize: number = 10;
+  private symbolRotation: number = 0;
```

Comment:
These two values are also used in `SymbolDrawingToolInputComponent` and are probably related, no? If so, these should be configuration values. Given that we're not in the angular context here, you could use `/shared/constants` for that.